package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "playcanvas",
   "version": "2.1.0-dev",
-  "lockfileVersion": 3,
```

Comment:
Looks like you're on an old version of NPM. The `package-lock.json` has been bumped down to 2 from 3. This adds about 7.500 lines to the file!

Reply:
Ouch

Reply:
Hmm, updated npm to 10.8.3 and getting warnings now. Do you have the same? What version do you use?

![image](https://github.com/user-attachments/assets/cba80607-c54b-48e7-8aea-49bf4fd8c709)

Reply:
I'm using 9.7.1. But those warnings are fine. The `eslint-plugin-import` team are working to add support for ESLint 9. I think they'll be done in a week or two. Then, those warnings will go away.